search/sort.go

```diff
@@ -21,12 +21,13 @@ import (
 	"math"
 	"sort"
 	"strings"
+	"unicode/utf8"
 
 	"github.com/blevesearch/bleve/v2/geo"
 	"github.com/blevesearch/bleve/v2/numeric"
 )
 
-var HighTerm = strings.Repeat(string([]byte{0xff}), 10)
+var HighTerm = string(utf8.MaxRune)
```

Comment:
one thought is that we repeat the max rune 3x, that way get 12 bytes (approximately as long as the byte sequence we had before)

Reply:
Ok, sounds good.